packages/sources/ncfx/src/endpoint/crypto.ts

```diff
@@ -4,8 +4,18 @@ import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
 export const supportedEndpoints = ['crypto']
 
 export const inputParameters: InputParameters = {
-  base: ['base', 'from', 'coin'],
-  quote: ['quote', 'to', 'market'],
+  base: {
+    aliases: ['from', 'coin'],
+    description: 'The symbol of the currency to query',
+    options: ['BTC', 'ETH', 'USD'],
+    required: true,
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    description: 'The symbol of the currency to convert to',
+    options: ['BTC', 'ETH', 'USD'],
```

Comment:
This endpoint supports many more symbols than this. We should either pull an accurate list from the data provider or leave this off.

CC: @austinborn

Reply:
Actually, this is just used for the README huh? Not actually enforced?

Reply:
> Actually, this is just used for the README huh? Not actually enforced?

no, enforced. if there is 'options', then values should match

Reply:
Yeah it's enforced. If these are only a sample of options, then it should go in the description section